virttest/qemu_monitor.py

```diff
@@ -2536,6 +2567,45 @@ def blockdev_del(self, node_name):
         args = {"node-name": node_name}
         return self.cmd(cmd, args)
 
+    def blockdev_backup(self, device, target, sync, job_id=None, **kwargs):
+        """
+        Backup block device via QMP command blockdev-backup
+
+        :param string device:  device node-name
+        :param string target: target device node-name
+        :param string sync: sync mode, vaules are 'top', 'full', 'none', 'incremental'
+        :param dict kwargs: dict for other options, eg. 'comparess', 'on-source-error'
+        :return: block job ID
+        :rtype: string
+        """
+        cmd = "blockdev-backup"
+        self.verify_supported_cmd(cmd)
+        args = copy.copy(kwargs)
+        if not job_id:
+            job_id = utils_misc.generate_random_id()
+        args.update({"device": device, "target": target,
+                     "sync": sync, "job-id": job_id})
+        self.cmd(cmd, args)
+        return job_id
+
+    def _operate_block_job(self, sub_cmd, identifier, **kwargs):
+        """
+        Block job operator
+
+        :param string sub_cmd: supported sub-commands are: 'cancel', 'pause', 'resume',
+                               'set-speed', 'complete', 'dismiss', 'finalize'
+        :param string identifier: identifier for block job, it should be job ID or device ID
+        :param dict kwargs: dict of block job options
+        :return: nothing on successful
+        """
+        self.verify_supported_cmd(sub_cmd)
+        if sub_cmd in ["job-dismiss", "job-finalize"]:
+            args = {"id": identifier}
```

Comment:
For these two commands, seems that need to add the `block-` prefix, right?

Reply:
No, it's raw QMP command.

Reply:
Do you have the test log for this?

Reply:
2019-06-27 10:51:31: {"execute": "job-dismiss", "arguments": {"id": "idEFRNBm"}, "id": "XGHCg9BL"}
2019-06-27 10:51:31: {"timestamp": {"seconds": 1561603891, "microseconds": 998136}, "event": "JOB_STATUS_CHANGE", "data": {"status": "null", "id": "idEFRNBm"}}
2019-06-27 10:51:31: {"return": {}, "id": "XGHCg9BL"}

Reply:
OK, thanks for your information. The problem is clear to me now, QEMU has two QMP commands for the `dismiss` action: `block-job-dismiss` and `job-dismiss`. Although I'm still not sure which is the one we wanted, according to the method name `_operate_block_job` you are introducing, `block-job-dismiss` would be the more suitable one to me.

To avoid misleading, I'd suggest the following approach:
```python
def dismiss_block_job(self, ...):
    # call `block-job-dismiss`
    ...

def job_dismiss(self, ...):
    # call `job-dismiss`
    ...
```
how do you think about that?